src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imaginginterface.py

```diff
@@ -46,6 +47,8 @@ def __init__(
         assert tif_file_paths, f"The TIF image files are missing from '{self.folder_path}'."
         self.image_metadata = extract_extra_metadata(file_path=tif_file_paths[0])
         super().__init__(folder_path=folder_path, channel_name=channel_name, plane_name=plane_name, verbose=verbose)
+        if stub_test:
+            self.imaging_extractor=self.imaging_extractor.frame_slice(start_frame=0, end_frame=100)
```

Comment:
This is definitely a workaround, but have you investigated why `stub_test` doesn't work in the parent `BaseImagingExtractorInterface`?

https://github.com/catalystneuro/neuroconv/blob/87552f3de5f40f879c204d1e41a9ddeef0967aea/src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py#L116